### ISSUE TITLE
problem in https://dharmalekha.info/texts/INSIDENKNglumbangDungik

### ISSUE THREAD

**Issue by arlogriffiths** [erc-dharma/tfc-nusantara-epigraphy#75]
dear @michaelnmmeyer — do you understand why the `<bibl>` items are not displayed?

**Comment by michaelnmmeyer** [comment]
This is not working because `@target` attributes are missing a `bib:` prefix:

    <ptr target="TitiSurtiNastiti+al2024_01"/>

... should be:

    <ptr target="bib:TitiSurtiNastiti+al2024_01"/>

That said, I might be able to supply it when missing. I will look at it.

**Comment by arlogriffiths** [comment]
thanks a lot. problem solved.

in my opinion, we need to remain strict about correct encoding, and ideally the schema would supply a message for a case like this, reminding the encoder of the need to prefix bib: so we can abvoid bothering you. I prefer to minimize the number of cases where our system patches encoding errors, rather it should render them visible and if possible propose solutions in the form of messages.

@danbalogh probably has an opinion on the matter.

**Comment by danbalogh** [comment]
I completely agree with Arlo and think that encoding practice should aim to be as clean as possible and patches should be avoided. That is to say, I would like to keep to the minimum the number of automatic edits and corrections in our XML files. If, on the other hand, the schema can be made intelligent enough to understand the contents of ALL `bibl/ptr/@target` as beginning with "bib:", so that we can dispense of the bib: prefixes altogether (and not need separate software to hard-code those into the file), then I would be amenable to that, since it simplifies the encoding. But I don't think this is important enough to bother.

**Comment by arlogriffiths** [comment]
Thanks. If the eradication of "bib:" is possible, then eradicating "part:" may also be possible. If I am not mistaken, the contexts of usage are never the same.
